src/main/java/seedu/pettracker/storage/Storage.java

```diff
@@ -236,15 +277,25 @@ private String getWeight(String line) {
         return weight;
     }
 
-    private String getTaskName(String line) {
+    private String getTaskName(String line) throws EmptyTaskNameException {
         String[] words = line.split("\\|", 3);
         String taskName = words[1];
+
+        if (taskName.trim().isEmpty()) {
+            throw new EmptyTaskNameException();
+        }
+
         return taskName;
     }
 
-    private String getTaskStatus(String line) {
+    private String getTaskStatus(String line) throws InvalidMarkTaskSymbolException {
         String[] words = line.split("\\|", 2);
         String taskStatus = words[0];
+
+        if (!taskStatus.equals("1") && !taskStatus.equals("0")) {
```

Comment:
This is great, our data corruption manual testing will now throw errors if its not 1/0